jordan_bundy/lib/greet.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var exports = module.exports = {};
+
+var greet = function greet(name) {
```

Comment:
Good to know!  I had a previous class where our instructor preferred us to use both, but I thought it seemed strange.